src/app/server/Mdns.h

```diff
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <core/CHIPError.h>
+#include <messaging/ReliableMessageProtocolConfig.h>
```

Comment:
Is it used in this file? Or just `Mdns.cpp`?

Reply:
Only in Mdsn.cpp, I moved it.